src/command.rs

```diff
@@ -232,6 +233,7 @@ pub(crate) fn parse_commandline_args(
                 #[cfg(feature = "plugin")]
                 plugins,
                 no_config_file,
+                no_defaults,
```

Comment:
do we really need another parameter? can't we just get by with --no-config-files which includes config files and defaults?

Reply:
I didn't have it in there initially, but then I was going through #13671 and you had mentioned that someone might *want* to continue to use the monolithic config.  Without the ability to disable defaults, they would incur double-loading time that you pointed out there.  It's about a 5ms hit on my system.  It's not huge, and I expect most people will transition to the "override" form (especially if that's  what we promote in the Doc), but I figured it was a "nice-to-have".

Alternative - And perhaps better.  If they have a `.no-defaults` file in the config directory, then we don't load defaults.  This would be a more "permanent" solution for those who wanted to use the monolithic config. I considered that after I was pretty much done with putting in the flag.

Thoughts?

Reply:
It's been removed (see comments below), so marking resolved.